README.md

```diff
@@ -23,140 +25,154 @@
-`/etc/skel/.bashrc` everytime a user is connected. The purpose of the script is similar - it will download and
-merge the kubeconfigs for OPS and will get `kubectx` tool on the jumpbox.
+- put `export KUBECONFIG=[path-to-repo]/content-k8s-auth-setup/kubeconfig`  in `~/.zshrc`
+- execute `source cluster-login.zsh udde`
 
-#### Step-by-Step guide how to connect to EKS clusters
+## EKS
 
-1. Connect to Restricted VPN
-1. Checkout the kubectl-login repo
+### Info
 
-```shell
-git clone git@github.com:Financial-Times/kubectl-login.git
-```
+When provisioning a new EKS cluster the provisioning script will upload the
+corresponding kubeconfig to S3 bucket named `upp-kubeconfig-ACCOUNT-NUMBER`.
+In order to use it you need to get this kubeconfig and merge it with the
+kubeconfig files of the rest of EKS clusters. This way you will have a single
+united kubeconfig for all the EKS clusters.
 
-1. Get in `update-eks-kubeconfig/` folder
+If you want to do that automatically, there is this script
+`./update-eks-kubeconfig/update-eks-kubeconfig.sh`.
+The script will do the merge for you. It will store the merged
+`eks-kubecofig` under `$HOME/.kube`.
+More details on that in the #step-by-step guide below.
 
-```shell
-cd update-eks-kubeconfig/
-```
+#### Ops specifics
 
-1. Edit `update-eks-kubeconfig.sh` script and fill in the EKS cluster names in PROD and TEST accounts
+The script `ops-eks-kubeconfig.sh` inside `update-eks-kubeconfig` directory
+is intended to be used from OPS on the jumpbox (Upp Jumpbox p).
+The script sits in `/usr/local/bin` and is invoked by `/etc/skel/.bashrc`
+everytime a user is connected. The purpose of the script is similar - it will
+download and merge the kubeconfigs for OPS and will get `kubectx` tool
+on the jumpbox.
 
-```shell
-PROD_ACCOUNT_CLUSTERS=(
-  eks-pac-staging-eu
-  eks-pac-staging-us
-
-)
-TEST_ACCOUNT_CLUSTERS=(
-  eks-publish-staging-eu
-  eks-delivery-staging-eu
-)
-```
+### Connect to EKS cluster (Step by step guide)
 
-1. Execute `update-eks-kubeconfig.sh`
+1. Checkout the kubectl-login repo and get into the proper folder:
 
-```shell
-bash update-eks-kubeconfig.sh
-```
+    ```shell
+    git clone git@github.com:Financial-Times/kubectl-login.git
+    cd update-eks-kubeconfig/
+    ```
 
-Restricted VPN is now no longer needed.
+1. Check the script `update-eks-kubeconfig.sh` and update the EKS cluster names
+  for PROD and TEST accounts, if needed:
 
-1. Export KUBECONFIG
+    ```shell
+    PROD_ACCOUNT_CLUSTERS=(
+      ...
+    )
+    TEST_ACCOUNT_CLUSTERS=(
+      ...
+    )
+    ```
 
-```shell
-export-eks-kubeconfig
-```
+    How to check what are the EKS clusters' names?
 
-or if you miss the alias
+    - Log into the [AWS console](https://awslogin.in.ft.com/adfs/ls/IdpInitiatedSignOn.aspx?loginToRp=urn:amazon:webservices).
+    - Check accordingly test and prod accounts.
+    - Look for `EKS` and check the cluster names there.
```

Comment:
A link to the EKS console in AWS instead of the link to the login page may be more useful.

Reply:
I do not see the advantage of having a direct link, because the link is region specific and it is skipping the part which account is being used (either test or prod) to gather all EKS cluster names.
However I've altered the instruction in the last step and added it in there, since someone is finding it easier this way.